docs/data/charts/components/components.md

```diff
@@ -87,6 +87,29 @@ For example, you can use `useLineSeries` to obtain the series of a line chart an
 
 {{"demo": "SeriesDemo.js"}}
 
+### Animation
+
+Some elements of charts are animated, such as the bars in a bar chart or the slices in a pie chart.
+
+We prioritize CSS animations, but some animations aren't possible using CSS. In those cases, we use JavaScript to animate elements.
+
+The animations of elements that are animated using CSS can be customized by overriding the CSS classes:
+
+{{"demo": "CSSAnimationCustomization.js"}}
+
+When it isn't possible to leverage CSS animations, the default components are animated using custom hooks.
+
+If you want to use the default animations in custom components, you can use these hooks.
+They are available for each element that is animated using JavaScript and are prefixed with `useAnimate`, e.g., `useAnimateLine` or `useAnimatePieArcLabel`.
+
+In the example below, labels are positioned above the bars they refer to and are animated using the default animation:
+
+{{"demo": "JSAnimationCustomization.js"}}
+
+Alternatively, you can use own our animation library to create custom animations, such as React Spring:
+
```

Comment:
Maybe comment on the performance hit of using js animation libraries?

```suggestion

Note that JS animation libraries can cause performance issues when rendering many data points or relying on interactions.

```

Reply:
I guess we can add this, but what do you mean by "relying on interactions"?

Reply:
I wanted to mean that animations have the possibility of making user interactions like highlights, tooltips and zoom slower

Reply:
Makes sense. Updated 👍